notebooks/sketchup_demo.ipynb

```diff
@@ -0,0 +1,171 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interop with SketchUp via `.dae` files\n",
+    "\n",
+    "In this notebook, we will explore how to use the `SketchUpModel` facade to interact with exportable tqec SketchUp models. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from tqec.sketchup.model import SketchUpModel\n",
+    "from tqec.sketchup.block import BlockType"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Library Blocks\n",
+    "\n",
+    "All the library/template blocks are defined in the backend to be the building blocks for a model. A library block is defined by its geometries(faces) and materials(X/Z/H face type). For example, a `zxx` block is represented by its 6 faces as follows:\n",
+    "```txt\n",
```

Comment:
Put ````python`  instead of ````txt` for readability in VS Code.

Reply:
I don't have the problem in my IDE but I can make the change.